Gemfile

```diff
@@ -66,6 +66,7 @@ if RUBY_ENGINE == "ruby"
   gem 'commonmarker', '~> 0.20.0'
   gem 'pandoc-ruby', '~> 2.0.2'
   gem 'simplecov', require: false
+  gem 'ruby2_keywords', '~> 0.0.4'
```

Comment:
The Gemfile only holds our test dependencies, this will be a runtime dependency for Sinatra, so it needs to be in the gemspec: https://github.com/sinatra/sinatra/blob/cf404526373298f06a59e1a129b6cdf8282ae216/sinatra.gemspec#L46-L49